docs/application/dotnet/get-started/overview.md

```diff
@@ -1,33 +1,35 @@
 # Get Started with .NET Applications
 
-When you create your first Tizen .NET application, you can select from the following options that uses C# as the programming language:
+When you create your first Tizen .NET application, you can select from the following options that use C# as the programming language:
```

Comment:
```suggestion
When you create your first Tizen .NET application, you can build your applications using C# language as follows:
```

Reply:
Based on your suggestion in https://github.com/Samsung/tizen-docs/pull/1130#discussion_r444121718, and to maintain consistency across all the get started pages, can we rephrase as:

You can create Tizen .NET applications in C#. You can also use the Tizen .NET APIs to access the Tizen platform-specific features. To create your first Tizen .NET application, you can select from the following profiles:

Please verify and share your valuable feedback.